Tasks/AzureKeyVaultV1/operations/KeyVault.ts

```diff
@@ -192,29 +192,31 @@ export class KeyVault {
             return;
         }
 
+        let secretNameWithoutVerson = secretName.split("/")[0];
```

Comment:
`secretNameWithoutVersion` (spelling mistake)
Add a comment explaining why you're doing this here.

Reply:
You could also just override the existing `secretName` variable because it will make sense with the comment.

Reply:
@ds-ms Fixed the spelling - Keeping the new variable to enable us to write both original name and name without identifier.